### PR TITLE
yast: Update SLE versions to SP2 for Cloud 7

### PR DESCRIPTION
**Pull request by sidthe**

### CODE REVIEW

**Comment by jsuchome** [comment]
Please update the `yast2-crowbar.changes` and bump the version

**Comment by sidthe** [comment]
@jsuchome done

File: package/yast2-crowbar.changes [hunk 1]
```diff
@@ -72,7 +78,7 @@ Wed Oct  7 13:52:52 CEST 2015 - jsuchome@suse.com
 
 - Adapt module so it can be called during first stage of installation
   (which is the only stage in SLES12)
-- 3.1.8 
+- 3.1.8
```
Comment (jsuchome) [inline package/yast2-crowbar.changes]: Please do not adapt the old items in changelog, build service will not like it

Comment (jreidinger) [inline package/yast2-crowbar.changes]: this small changes are fine, it is usually ignored, also typos, but you should not rephrase it nor change it too much

Comment (sidthe) [inline package/yast2-crowbar.changes]: yeah, my linter is cleaning up the trailing spaces, but I believe that is fine, right?

Comment (jreidinger) [inline package/yast2-crowbar.changes]: yeah, it should be. We do it also sometimes and never rejected

File: src/include/crowbar/complex.rb [hunk 2]
```diff
@@ -84,14 +84,14 @@ def initialize_crowbar_complex(include_target)
...
       # arch value for currently selected repository
       @current_arch = "x86_64"
 
       @platform2label = {
         # target platform name
-        "suse-12.1" => _("SLES 12 SP1")
+        "suse-12.2" => _("SLES 12 SP2")
       }
 
       @repos_location         = ""
```
Comment (vuntz) [inline src/include/crowbar/complex.rb]: It'd be good to change the examples on line 118/119 if possible.

**Comment by sidthe** [comment]
@vuntz improved the examples. In my understanding those are now the right paths

**Comment by vuntz** [comment]
Cool, thanks, still +1.

@jsuchome okay to merge?

**Comment by jsuchome** [comment]
I'm fine with the change, just wondering that we probably did not create a branch for Cloud6....

**Comment by vuntz** [comment]
> just wondering that we probably did not create a branch for Cloud6....

I see a Cloud6 branch, actually :-)

**Comment by vuntz** [comment]
Oh, but that branch seems awfully outdated...

File: package/yast2-crowbar.changes [hunk 3]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Nov 21 10:11:58 UTC 2016 - rarcea@suse.com
+
+- Update SLE versions to SP2 for Cloud 7
+- 3.1.20
```
Comment (vuntz) [inline package/yast2-crowbar.changes]: After discussing with @jsuchome, can you bump to version 3.2.0? We'll keep 3.1.x for Cloud 6.

Comment (sidthe) [inline package/yast2-crowbar.changes]: done

**Pull request merged**